lib/register.js

```diff
@@ -35,7 +47,9 @@ async function register(req, res) {
     contact = `sip:${uri.user}@${req.source_address}:${req.source_port}`;
     contactHdr = contactHdr.replace(/expires=\d+/, `expires=${expires}`);
   }
-  await registrar.add(registration.aor, contact, protocol, expires);
```

Comment:
I think we should try to always wrap an `await` call in a try-catch block.  The catch handles the case of a rejected Promise.